src/main/java/org/folio/gobi/Mapper.java

```diff
@@ -552,8 +552,7 @@ private void mapPurchaseOrderLine(List<CompletableFuture<?>> futures,CompositePo
       futures
```

Comment:
This block of code needs to be removed, we no longer have po line workflow status